pkg/aws/ec2helper_test.go

```diff
@@ -15,22 +15,19 @@ func TestGetRouteTableIDs(t *testing.T) {
 		name     string
 		EC2API   *MockEC2API
 		VpcID    string
-		expected RouteTableIDs
+		expected []string
 	}{
 		{
-			name:   "Returns both a public and private route table id",
-			EC2API: &MockEC2API{},
-			VpcID:  "test-vpc-id",
-			expected: RouteTableIDs{
-				Public:  "PublicRouteTableId",
-				Private: "PrivateRouteTableId",
-			},
+			name:     "Returns both a public and private route table id",
+			EC2API:   &MockEC2API{},
+			VpcID:    "test-vpc-id",
+			expected: []string{"PublicRouteTableId", "PrivateRouteTableId"},
```

Comment:
I recommend changing the MockEC2API DescribeRouteTables function to return `RouteTableId1` and `RouteTableId2` rather than `PublicRouteTableId` and `PrivateRouteTableId` to avoid confusing future developers since this change kinda does away with the contexts of "public" and "private" route tables.

You may want to consider adding a third `RouteTableId` into the tests to make sure the code behaves as expected when there are more than 2 route tables associated with the VPC.